.github/workflows/release.yml

```diff
@@ -0,0 +1,93 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version'
+        required: true
+
+jobs:
+  release:
+
+    runs-on: ubuntu-latest
+    container: adoptopenjdk:11-jdk-hotspot-bionic
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 15
+        uses: actions/setup-java@v1
+        with:
+          java-version: 15.0.1
+          java-package: jdk+fx
```

Comment:
JDK15 collides with the container description above using JDK11. does that work?

Reply:
Nope this is just a victim of copy and paste nad not being able to test actions before they are on develop

Reply:
should be better now